packages/metro/src/lib/__tests__/contextModuleTemplates-test.js

```diff
@@ -49,4 +49,15 @@ describe('getContextModuleTemplate', () => {
 
     expect(template).toMatchSnapshot();
   });
+
+  test('creates posix paths on windows for sync template', () => {
+    jest.resetModules();
+    jest.mock('path', () => jest.requireActual('path').win32);
+    const { getContextModuleTemplate: getWindowsTemplate } = require('../contextModuleTemplates');
```

Comment:
nit: run prettier-format
```suggestion
    const {
       getContextModuleTemplate: getWindowsTemplate,
     } = require('../contextModuleTemplates');
```

Reply:
Done